spec/features/find/switcher_cycles_spec.rb

```diff
@@ -94,6 +98,6 @@ def then_i_should_see_the_applications_closed_text
   end
 
   def and_i_should_see_the_correct_previous_recruitment_cycle_year
-    expect(page).to have_text("Previous cycle year#{RecruitmentCycle.current.year}") # After find reopens, the previous cycle year for the fake cycle becomes the current cycle year for the real cycle
+    expect(page).to have_text('Previous cycle year2023') # After find reopens, the previous cycle year for the fake cycle becomes the current cycle year for the real cycle
```

Comment:
Isn't this going to fail next year?

Reply:
Oh I see, you've frozen time so that will always be the year?

Reply:
Yeah we should tidy those up too. I feel like it might not be better to freeze time, but lets just get this in for now!